tests/view/test_reservations.py

```diff
@@ -16,6 +18,27 @@
 
 """
 
+def test_reservation_list(client, db_session):
+    employee = factories.EmployeeFactory(
+        company=factories.CompanyFactory()
+    )
+
+    with client.session_transaction() as session:
+        session["user_id"] = employee.id
+
+    #Cheated reservations
+    factories.ReservationFactory()
+    factories.ReservationFactory()
+    factories.ReservationFactory()
+    factories.ReservationFactory()
+
+    response = client.get("/reservations/teste/")
```

Comment:
@ebercamargo Why `teste`? Also let's use `flask.url_for` instead of hardocded urls

Reply:
I adjusted this.